src/Tests/Microsoft.Diagnostics.Monitoring.ConfigurationSchema/SchemaGenerator.cs

```diff
@@ -16,11 +16,24 @@ public string GenerateSchema()
             schema.Id = @"https://www.github.com/dotnet/dotnet-monitor";
             schema.Title = "DotnetMonitorConfiguration";
 
+            //Allow other properties in the schema.
+            schema.AdditionalPropertiesSchema = JsonSchema.CreateAnySchema();
+
             //HACK Even though Properties is defined as JsonDataExtension, it still emits a property
             //called 'Properties' into the schema, instead of just specifying additionalProperties.
             //There is likely a more elegant way to fix this.
             schema.Definitions[nameof(EgressProvider)].Properties.Remove(nameof(EgressProvider.Properties));
 
+            //TODO Figure out a better way to add object defaults
```

Comment:
Probably could create a ISchemaProcessor that applies this default to any property that is an object and doesn't have a default value. But you don't have to do this now.